src/routes/resources/kendo-japanese-pronunciations-and-definitions/+page.svelte

```diff
@@ -0,0 +1,759 @@
+</dl>
+
+<h3>Concepts</h3>
+<dl>
+	<dt>Zanshin: 残心 【ざんしん】</dt>
+	<dd>
+		(‘kokoro wo nokosu’- I leave my spirit behind, ‘kokoro ga nokoru’ – my spirit remains) follow
+		through.
+	</dd>
+
+	<dt>Seme: 攻め 【せめ】</dt>
+	<dd>Attack</dd>
+
+	<dt>Sutemi: 捨て身; 捨身 【すてみ】</dt>
+	<dd>
+		(lit. To throw away one’s life) At the risk of one’s life. Without reservation, total
+		commitment.
+	</dd>
+
+	<dt>Ki ken tai icchi 気剣体一 致【きけんたいっち】</dt>
+	<dd>
+		(lit. the spirit, sword, body are one.) Can mean that there is no true distinction between the
+		spirit, sword, body and mind in kendo.
+	</dd>
+
+	<dt>Enzan no metsuke 遠山の目付け【えざんのめつけ】</dt>
+	<dd>
+		(lit: looking as if gazing at a far away mountain) To see everything in proper perspective, not
+		focusing on a particular detail to the point of not seeing something else.
+	</dd>
+
+	<dt>Kiai 気合い【きあい】</dt>
+	<dd>
+		Showing of one’s fighting spirit, drawing from the “hara” or center of ones being. Sometimes
+		used to goad an opponent on.
+	</dd>
+
+	<dt>Mitsu no sen 三つの先【みつのせん】</dt>
+	<dd>“The three advantageous positions”</dd>
+
+	<dt>Sen sen no sen 先々の先【せんせんのせん】</dt>
+	<dd>
+		To put oneself at an advantage by preceiving one’s opponent’s intention, and striking before the
+		opponent has a chance to realize their attack (ie. debana waza).
+	</dd>
+
+	<dt>Sen 先【せん】</dt>
+	<dd>
+		To take an opponent’s initiative after they have begun an attack (nuki waza). This strategy is
+		exemplified in the first and second kata.
+	</dd>
+
+	<dt>Tai no sen 待の先【たいのせん】also Go no sen 後の先 【ごのせん】</dt>
+	<dd>
+		“Wait” or “after”. To endure an opponent’s attack and counterattack after the opponent’s
+		momentum is gone.
+	</dd>
+</dl>
+
+<h2>A very short guide to pronunciation:</h2>
+<p>
+	The following table is a “quick” pronunciation guide to Japanese characters if you are confused
+	about how a term is pronounced. There really is no replacement for hearing someone speak the terms
+	correctly, so please use this portion of the glossary as a very rough guide.
+</p>
+<p>
+	Japanese letters other than vowels are made up of two sounds, a hard consonant and a vowel sound.
+	Japanese vowels tend to be “short vowel” sounds in English. Each column in the table corresponds
+	to one particular consonant, and each row of the table corresponds to a certain vowel sound. The
+	first column is made up of all vowel sounds. Several spaces in the table are left blank because
+	several letters in the Japanese alphabet have been eliminated. This table uses only “hiragana”
+	characters, because almost all of the terms in this list use this character set. A separate
+	character set called “katakana” exists for words from other languages.
+</p>
+<table dir="ltr">
+	<tr>
+		<td> </td>
+		<th scope="col">“K”</th><th scope="col">“S”</th><th scope="col">“T”</th><th scope="col">“N”</th>
+		<th scope="col">“H”</th><th scope="col">“M”</th><th scope="col">“Y”</th><th scope="col">“R”</th>
```

Comment:
It's strange to me to have multiple but not all on the same line. I would probably do one line per cell but I guess it doesn't supper matter

Reply:
I was doing multiple to make it easier to follow if you wanted to change a certain cell. Like this, you can recognize which column its in and what the other cells in that column contain just by looking at how the th's and td's are arranged (ex, row 0, col 1 in the code). But honestly the content in these tables will likely only be rearranged or formatted so that's not particularly useful. I'll change it tomorrow